src/stcal/jump/jump.py

```diff
@@ -514,6 +532,21 @@ def detect_jumps(
     # Return the updated data quality arrays
     return gdq, pdq, total_primary_crs, number_extended_events, stddev
 
+# Find micrometeorite flashes that light up the entire array in a small number
+# of groups.  Do this by looking for cases where greater than mmflash_pct percent
+# of the array is flagged as 'JUMP_DET' and flag the entire array.
+def find_micrometeorite_flashes(gdq, jump_flag, mmflashfrac):
+    npix = gdq.shape[2]*gdq.shape[3] # Number of pixels in array
+    # Loop over integrations
+    for ii in range(0,gdq.shape[0]):
+        # Loop over groups
+        for jj in range(0,gdq.shape[1]):
+            indx = np.where(gdq[ii,jj,:,:] & jump_flag != 0)
+            fraction = float(len(indx[0])) / npix
+            if (fraction > mmflashfrac):
+                gdq[ii,jj,:,:] = gdq[ii,jj,:,:] | jump_flag
+                log.info("Detected a micrometeorite flash in integ = %i, group= %i", ii, jj)
```

Comment:
Why not use an f-string?

Reply:
If that's the convention I certainly can- done.